app/test_split_integer.py

```diff
@@ -1,21 +1,26 @@
+import pytest
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert split_integer(6, 1) == [6]
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(8, 2) == [4, 4]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
I think that this test should have 1 as the second parameter. What do you think?